ffi/capi/src/locale_core.rs

```diff
@@ -102,7 +102,7 @@ pub mod ffi {
         /// Set the region part of the [`Locale`].
         #[diplomat::rust_link(icu::locale::Locale::try_from_str, FnInStruct)]
         #[diplomat::rust_link(icu::locale::Locale::try_from_utf8, FnInStruct, hidden)]
-        #[diplomat::attr(all(*, not(dart)), setter = "region")]
+        #[diplomat::attr(all(supports = setters, not(dart)), setter = "region")]
```

Comment:
yep, sorry, should have told you this, forgot that `auto` doesn't work inside `all`